homeassistant/components/wilight/config_flow.py

```diff
@@ -2,20 +2,19 @@
 from urllib.parse import urlparse
 
 import pywilight
+from pywilight.const import DOMAIN  # pylint: disable=unused-import
```

Comment:
We don't want the library to know the implementation details of Home Assistant (specifically that the integration is named `wilight` in Home Assistant)

This value should come from the integration itself since its a property of the integration.

Reply:
Changed to DOMAIN = "wilight"